### PR TITLE
Update readme: Add new project status

### PR DESCRIPTION
**Pull request by nebuk89**
Updating the main toolbox readme to reflect the 2020 status of the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+
+Docker released the Docker Toolbox project to make it easier for developers who work on Mac and Windows to get started using Docker. In 2016 Docker released Docker Desktop which superseded toolbox and was significantly easier for the majority of users to get started.  This still left some users behind, predominantly users who were on Windows Home editions, Windows 7, Windows 8 and users of Virtualbox. 
+Since 2016 there have been a number of changes. Windows 7 is no longer supported and the mainstream support of Windows 8.1 has ended. The majority of Windows users now on a version of Windows 10.  Since [Virtualbox 6.0](https://docs.oracle.com/en/virtualization/virtualbox/6.0/admin/hyperv-support.html#:~:text=Oracle%20VM%20VirtualBox%20can%20be,engine%20for%20the%20host%20system) users have been able to run virtualbox and HyperV at the same time on their Windows machines, allowing users to use virtualbox and Docker Desktop side by side on HyperV. For Windows Home users, WSL 2 is available and Docker Desktop now uses this to provide [Desktop for Windows Home](https://docs.docker.com/docker-for-windows/install-windows-home/)
```
Comment (mat007) [inline README.md:6]: typo: missing `are` or `run` or … before `now on a version of Windows 10`?

Comment (mat007) [inline README.md:6]: typo: `virtualbox` -> `VirtualBox`?
typo: `HyperV` -> `Hyper-V`?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
 Docker Toolbox
+
+### TL/DR: Docker will be ceasing Support and Development of the Docker Toolbox project in 2020 - going forward please use [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+Docker released the Docker Toolbox project to make it easier for developers who work on Mac and Windows to get started using Docker. In 2016 Docker released Docker Desktop which superseded toolbox and was significantly easier for the majority of users to get started.  This still left some users behind, predominantly users who were on Windows Home editions, Windows 7, Windows 8 and users of Virtualbox.
```
Comment (mat007) [inline README.md:5]: typo: `virtualbox` -> `VirtualBox`?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,13 @@
 Docker Toolbox
+
+### TL/DR: Docker will be ceasing Support and Development of the Docker Toolbox project in 2020 - going forward please use [Docker Desktop](https://www.docker.com/products/docker-desktop)
```
Comment (mikeparker) [inline README.md:3]: I think the readme should state the current status not the future state. I'd suggest something instead along the lines of:
> This project and repository is now deprecated and is no longer in active development. Please use Docker Desktop instead where possible.

Or a simpler change to use present tense:
> Docker has ceased support and development of the Docker Toolbox project.

Comment (mikeparker) [inline README.md:3]: I'd be tempted to title this `Deprecation Notice` instead of `TL/DR`

Comment (mikeparker) [inline README.md:3]: Maybe add some relevant emoji to draw attention ⚠️ or ❗ or 🔴 or 🛑

File: README.md (hunk 1 above)
Comment (nebuk89) [reply README.md]: Fixed

Comment (nebuk89) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (nebuk89) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (nebuk89) [reply README.md]: Took the first suggestion

Comment (nebuk89) [reply README.md]: Good idea, changed

Comment (nebuk89) [reply README.md]: Added

**Review by justincormack — APPROVED** [review]

**Pull request merged**